cli-tests/src/utils.rs

```diff
@@ -63,3 +65,10 @@ pub fn generate_mock_codeowners<T: AsRef<Path>>(directory: T) {
     "#;
     fs::write(directory.as_ref().join("CODEOWNERS"), CODEOWNERS).unwrap();
 }
+
+pub fn write_junit_xml_to_dir<T: AsRef<Path>>(xml: &str, directory: T) -> Result<()> {
+    let path = directory.as_ref().join("junit-0.xml");
+    let mut file = fs::File::create(&path)?;
+    file.write_all(xml.as_bytes())?;
+    Ok(())
+}
```

Comment:
No need to use `anyhow` or return a result here. `.unwrap()` is fine in tests since we want hard failures anyways